examples/pylint/e4702_modified_iterating_dict.py

```diff
@@ -0,0 +1,5 @@
+data = {'x': 1, 'y': 2, 'z': 3}
+
+for key, value in data:
+    data['w'] = 0
```

Comment:
Good---please add a comment, similar to the other example files, indicating that the error occurs on this line.